### PR TITLE
Feature/jira custom field assignment

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

File: sdetools/alm_integration/alm_plugin_base.py [hunk 1]
```diff
@@ -210,11 +218,11 @@ def initialize(self):
         self.config.process_boolean_config('how_tos_in_scope')
         self.config.process_boolean_config('alm_standard_workflow')
         self.config.process_json_dict('alm_custom_fields')
+        self.config['alm_custom_fields_template'] = {k:v for k,v in self.config['alm_custom_fields'].items()}
 
         if self.feature_custom_lookup:
             self.config.process_json_dict('alm_custom_lookup_fields')
-
-        self.init_custom_fields()
+            self.config['alm_custom_lookup_fields_template'] = {k:v for k,v in self.config['alm_custom_lookup_fields'].items()}
```
Comment (emintham) [inline sdetools/alm_integration/alm_plugin_base.py]: You'd want `dict((k, v) for k, v in self.config['alm_custom_lookup_fields'].items())` for older versions of python.

Comment (geoffwhittington) [inline sdetools/alm_integration/alm_plugin_base.py]: This PR is out of date actually, this code is gone as of today. maybe I should close the PR...

Comment (emintham) [inline sdetools/alm_integration/alm_plugin_base.py]: That was a rather fast reply.....

**Comment by geoffwhittington** [comment]
Closing since this was covered by #280

**Pull request closed without merging**